CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+## How to contribute to ezplatform.com
```

Comment:
@SylvainGuittard in fact contributing.md is about the process, guidelines... and should point to official documentation.

It's not about installing the project.

I would really recommend to point to documentation to avoid copy/paste and then unsynchronize documentation.

Here is symofny contributing file for example : https://github.com/symfony/symfony/blob/master/CONTRIBUTING.md